README.md

```diff
@@ -1,2 +1,132 @@
+
+For the Support Steward, we track both the current and incoming team members as we have two people overlapping in this role.
+
+```json
+{
+    "meeting_facilitator": {
+        "incoming": {
+            "name": "display_name",
+            "id": "slack_id"
+        },
+        "outgoing": {
+            "name": "display_name",
+            "id": "slack_id"
+        }
+    },
+    "support_steward": {
+        "incoming": {
+            "name": "display_name",
+            "id": "slack_id"
+        },
+        "current": {
+            "name": "display_name",
+            "id": "slack_id"
+        },
+        "outgoing": {
+            "name": "display_name",
+            "id": "slack_id"
+        }
+    }
+}
+```
+
+## Scripts
+
+All scripts are written in Python and are located in the [`src`](src/) folder.
+
+### `get_slack_team_members.py`
+
+This script interacts with the Slack API to produce a dictionary of Slack users who are members of a given Slack team (formally called a "usergroup" in the API), and their IDs.
+The script requires two environment variables to be set:
+
+- `TEAM_NAME`: The name of the Slack team to list members of, e.g., `tech-team`
+- `SLACK_BOT_TOKEN`: A bot user token for a Slack App installed into the workspace.
+  The bot requires the `usergroups:read` and `users:read` permission scopes to operate.
+  It does not need to be a member of any channels in the Slack workspace.
+
+The script will generate a dictionary of members of `TEAM_NAME` where the keys are the users' display names, and the values are their associated user IDs.
+The dictionary is ordered alphabetically by its keys.
+
+### `set_team_roles.py`
+
+This script reads in [`team-roles.json`](#team-roles-json-file-structure) and the dictionary generated by [`get_slack_team_members.py`](#get_slack_team_memberspy) and generates the next team member to serve in a given role by iterating one place through the dictionary.
+The updated team roles are written back to the `team-roles.json` file.
+There are command line flags to determine which role is to be updated.
+
+**Command line usage:**
+
+```bash
+usage: set_team_roles.py [-h] [--update-meeting-facilitator] [--update-support-steward]
+
+Update our Team Roles by iterating through members of the Tech Team
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --update-meeting-facilitator
+                        Update the Meeting Facilitator role
+  --update-support-steward
+                        Update the Support Steward role
+```
+
+### `create_geekbot_standup.py`
+
+This script reads in [`team-roles.json`](#team-roles-json-file-structure) after it has been modified by [`set_team_roles.py`](#set_team_rolespy) and generates a Geekbot standup to notify the incoming team member for their upcoming role.
+
+The `MeetingFacilitatorStandup` broadcasts to the `team-updates` Slack channel, and the `SupportStewardStandup` broadcasts to the `support-freshdesk` channel.
```

Comment:
```suggestion
The `MeetingFacilitatorStandup` broadcasts to the `team-updates` Slack channel, and the `SupportStewardStandup` broadcasts to the `support-freshdesk` channel.
The [Geekbot app](https://geekbot.com/) needs to be installed to the Slack workspace and invited to the channels to which it will broadcast.
```